Cargo.toml

```diff
@@ -22,7 +22,7 @@ slotmap = "1.0.6"
 grid = { version = "0.9.0", optional = true }
 
 [features]
-default = ["std", "flexbox", "grid"]
+default = ["std", "flexbox", "grid", "taffy"]
```

Comment:
Can we call this `tree` rather than `taffy`? It seems a bit silly to have a feature flag called `taffy` in a crate called `taffy`!

Reply:
Yeah, I realise this is public API stuff, but I am curious how set you are on naming this default implementation Taffy, I already found it quite confusing! It could also be something a bit more explicit, like DefaultLayoutTree. Maybe the flag could be named `default_tree`?

Reply:
> I am curious how set you are on naming this default implementation Taffy

Not at all set. In fact it's on my radar to rename it before the next release. Only thing is deciding on a new name. `DefaultLayoutTree` could work. Other ideas I had include `SimpleTree` and `TaffyTree`. In any case, I think we might need to create an issue and allow for some discussion on this issue rather than just going ahead and changing it.

Reply:
Issue created https://github.com/DioxusLabs/taffy/issues/449

Reply:
I went with `taffy_tree` for now. Thought process: calling it `taffy` is confusing; the same is true for `tree` (since there's also `LayoutTree`). Right now it's called Taffy _and_ it's a tree.

And also, the flag should probably be renamed when `Taffy` gets renamed.